decidim-surveys/spec/system/registered_user_survey_spec.rb

```diff
@@ -47,14 +49,7 @@
     let!(:user) { create(:user, :confirmed, organization:) }
 
     before do
-      component.update!(
-        step_settings: {
-          component.participatory_space.active_step.id => {
-            allow_answers: true,
-            allow_unregistered: false
-          }
-        }
-      )
+      survey.update!(allow_answers: true, allow_unregistered: false)
```

Comment:
You can use the factories with traits for this instead of using the before block

Reply:
Done